services/npm/npm-unpacked-size.tester.js

```diff
@@ -1,27 +1,27 @@
-import { isFileSize } from '../test-validators.js'
+import { isMetricFileSize } from '../test-validators.js'
 import { createServiceTester } from '../tester.js'
 
 export const t = await createServiceTester()
 
 t.create('Latest unpacked size')
   .get('/firereact.json')
-  .expectBadge({ label: 'unpacked size', message: isFileSize })
+  .expectBadge({ label: 'unpacked size', message: isMetricFileSize })
 
 t.create('Nonexistent unpacked size with version')
   .get('/express/4.16.0.json')
   .expectBadge({ label: 'unpacked size', message: 'unknown' })
 
 t.create('Unpacked size with version')
   .get('/firereact/0.7.0.json')
-  .expectBadge({ label: 'unpacked size', message: '147 kB' })
+  .expectBadge({ label: 'unpacked size', message: '147.2 kB' })
 
 t.create('Unpacked size for scoped package')
   .get('/@testing-library/react.json')
-  .expectBadge({ label: 'unpacked size', message: isFileSize })
+  .expectBadge({ label: 'unpacked size', message: isMetricFileSize })
 
 t.create('Unpacked size for scoped package with version')
   .get('/@testing-library/react/14.2.1.json')
-  .expectBadge({ label: 'unpacked size', message: '5.41 MB' })
+  .expectBadge({ label: 'unpacked size', message: '5.4 MB' })
```

Comment:
This file is a good example of the difference between rounding to significant figures vs rounding to decimal places. Switching to 1 decimal place sometimes gives us more precision and sometimes less precision than 3 significant figures.